src/uu/mv/locales/en-US.ftl

```diff
@@ -61,6 +61,7 @@ mv-debug-skipped = skipped {$target}
 
 # Prompt messages
 mv-prompt-overwrite = overwrite {$target}?
+mv-prompt-overwrite-mode = replace {$target}, overriding mode{$mode_str} ({$perms})?
```

Comment:
missing space after mode ?

Reply:
This was my hack to address this: https://github.com/uutils/coreutils/pull/9599/files#diff-fc9209d88cfefac46cf90a30d3d80113d5afb85df15f646aae70d8c51d76b7c9R1257 I will try again to find a better way

Reply:
By formatting the permissions separately from the translate it appears to have mitigated the need for this hack